libnmstate/nm/applier.py

```diff
@@ -266,17 +266,38 @@ def _get_affected_devices(iface_state):
         iface_type = iface_state[Interface.TYPE]
         if iface_type == ovs.BRIDGE_TYPE:
             port_slaves = ovs.get_slaves(nmdev)
-            iface_slaves = [
-                iface for port in port_slaves for iface in ovs.get_slaves(port)
-            ]
+            iface_slaves = _get_ovs_interfaces(iface_state)
             devs += port_slaves + iface_slaves
         elif iface_type == LB.TYPE:
             devs += bridge.get_slaves(nmdev)
         elif iface_type == bond.BOND_TYPE:
             devs += bond.get_slaves(nmdev)
+
+        ovs_port_dev = ovs.get_port_by_slave(nmdev)
+        if ovs_port_dev:
+            devs.append(ovs_port_dev)
     return devs
 
 
+def _get_ovs_interfaces(iface_state):
+    """
+    Report a list of all interfaces that are connected to the OVS bridge.
+    This is a workaround solution for https://bugzilla.redhat.com/1797478.
+    """
+    ifaces = [
+        p[Interface.NAME]
```

Comment:
Why `p`? I think the variable name should be more precise..

Reply:
It enclosed on a list comprehension and very (very) close to what it iterates over.
Do you find it hard to read and understand?

Reply:
Done